core/src/test/java/org/apache/spark/JavaAPISuite.java

```diff
@@ -54,7 +54,7 @@
 import org.apache.hadoop.mapred.SequenceFileOutputFormat;
 import org.apache.hadoop.mapreduce.Job;
 import org.junit.After;
-import org.junit.Assert;
+import static org.junit.Assert.*;
```

Comment:
Although I dislike static imports in many cases, this is one instance where it can make sense, especially in helping shorten lines.